packages/sit-onyx/src/components/OnyxDataGrid/features/index.ts

```diff
@@ -0,0 +1,178 @@
+import { h, type Component, type WatchSource } from "vue";
+import type { DataGridRendererColumn, DataGridRendererRow } from "../../..";
+import type { DataGridEntry, DataGridMetadata } from "../types";
+
+export type TableFeature<TEntry extends DataGridEntry, TFeatureName extends symbol> = {
```

Comment:
```suggestion
export type DataGridFeature<TEntry extends DataGridEntry, TFeatureName extends symbol> = {
```

Isn't this about the data grid, not the table?

Reply:
Same for other docs/functions in this file, e.g. `useDataGridFeature` instead of `useTableFeature`

Reply:
done